app/app/api/roles/route.ts

```diff
@@ -0,0 +1,40 @@
+import { NextRequest, NextResponse } from 'next/server'
+import prisma from "database/db";
+
+const complete_view = {
+    name: true,
+    users: {
+        select: {
+            id: true
+        }
+    }
+}
+
+const partial_view = {
+    name: true,
+}
+
```

Comment:
Is the excess whitespace necessary here? Maybe also consider running prettier for your diff.